### DISCUSSION TITLE
Can you always deploy the latest Next.js version to Vercel right after announcement?

### DISCUSSION THREAD

**Discussion by syahzuan** [vercel/community#985] [Help] [answered]
Next.js 13 was just released hours ago, does that mean I could update to v13 and deploy to Vercel right away? I mean, some other hosts might specify they only support up to certain Next.js version, can't seem to find this info on Vercel website.

**Comment by amyegan** [comment, marked as answer]
Great question @syahzuan! Vercel supports Next 13 and you can are free to try out the latest features now. In fact, the [Next.js Boilerplate template](https://vercel.com/templates/next.js/nextjs-boilerplate) is already [using v13](https://github.com/vercel/vercel/blob/314a97b3187cf11e459cf9292f974ae7018a33d6/examples/nextjs/package.json#L12)!
